openedx/templates/vertical.html

```diff
@@ -119,7 +119,7 @@
         {% endif %}
         {% if elem["type"] == "video" %}
           {% if "html_content" not in elem %}
-            <video class="video-js vjs-default-skin" controls preload="auto" width="480px" height="270px" data-setup='{"autoplay": true, "preload": "true"}' >
```

Comment:
No need to go to 16 or 32 but add 0.75 and 1.25 please

Reply:
Sure, have made the changes.